totp/__init__.py

```diff
@@ -71,18 +100,21 @@ def copy_to_clipboard(text):
 
 
 def run():
-    pass_entry = get_pass_entry(sys.argv[1])
+    if sys.argv[1] == '-a':
+        add_pass_entry(sys.argv[2])
+    else:
```

Comment:
Maybe this whole `else` could be split off into a separate function too, for consistency's sake?